test/pipeline/pipeline.import.azure.master.json

```diff
@@ -658,7 +1184,7 @@
                         "enabled": true,
                         "continueOnError": false,
                         "alwaysRun": false,
-                        "displayName": "Check to see if this is a MOVE: PR",
+                        "displayName": "Is MOVE: PR?",
```

Comment:
now that we're done with the migration - we can probably just remove these tasks - this check for "MOVE:" and the two that ping tfitz and mumian that follow.